MC/run/PWGEM/runHFToDielectrons_pp.sh

```diff
@@ -21,13 +21,13 @@ NTIMEFRAMES=${NTIMEFRAMES:-1}
 RNDSIG=$((1 + $RANDOM % 100))
 echo $RNDSIG
 
-if [[ $RNDSIG -ge 0 && $RNDSIG -lt 40 ]];
+if [[ $RNDSIG -ge 0 && $RNDSIG -lt 20 ]];
```

Comment:
More a technicality (and not directly related to you changes): You compare via `$RNDSIG -ge 0`, however, `RNDSIG` will never be `0`.
Like this the probabilities for ccbar and bbar are not exactly the same. Or is this on purpose?